pkg/features/kube_features.go

```diff
@@ -888,7 +888,12 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 	DaemonSetUpdateSurge:                           {Default: true, PreRelease: featuregate.Beta},                    // on by default in 1.22
 	DownwardAPIHugePages:                           {Default: true, PreRelease: featuregate.Beta},                    // on by default in 1.22
 	AnyVolumeDataSource:                            {Default: false, PreRelease: featuregate.Alpha},
+<<<<<<< HEAD
```

Comment:
It‘s better not to have this special character. Can you rebase it and fix conflict in a single commit?
You can add the `co-author` in one commit.

Reply:
thanks alex. done.